ykllvmwrap/build.rs

```diff
@@ -58,12 +64,22 @@ fn main() {
     ccg.generate();
 
     // Ensure that downstream crates performing linkage use the right -L and -l flags.
-    let lib_dir = llvm_config().arg("--libdir").output().unwrap().stdout;
+    let lib_dir = Command::new(&ykllvm_bin("llvm-config"))
+        .arg("--link-shared")
+        .arg("--libdir")
+        .output()
+        .unwrap()
+        .stdout;
     let lib_dir = std::str::from_utf8(&lib_dir).unwrap();
     println!("cargo:rustc-link-search={}", lib_dir);
     apply_llvm_ld_library_path();
 
-    let libs = llvm_config().arg("--libs").output().unwrap().stdout;
+    let libs = Command::new(&ykllvm_bin("llvm-config"))
```

Comment:
How comes we removed `llvm_config()` as a function? I don't feel strongly, but it made sense to have one place to apply `--link-shared`.

Reply:
Partly a matter of taste, but it felt weird to have `llvm_config() -> Command` and then we'd still have to have `ykllvm_bin(...) -> Path` or similar.

Reply:
I guess I'm OK with it. We just have to remember to add `--link-shared` everywhere or we may statically link extra versions of llvm.